app/DoctrineMigrations/Version20160620142808.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Surfnet\StepupMiddleware\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20160620142808 extends AbstractMigration
```

Comment:
This does not seem right. Probably because my database has another state because of migrations of another branch.